tests/tasks/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from creator.organizations.factories import OrganizationFactory
+
+
+@pytest.fixture(autouse=True)
+def default_organization(db):
```

Comment:
It could be but it's clearer if you were to just inject `OrganizationFactory()` into the test so the reader can see it's a direct dependency. This would also go away if the new `permission_client` was used here as that creates an org for each client and user.